### PR TITLE
Update examples with latest from dev build

### PR DESCRIPTION
**Pull request by TerryHowe**
With these updated examples, the installs are working for me

```
% k get packages,packagebundles,packagebundlecontrollers -A
NAMESPACE       NAME                                         PACKAGE   AGE   STATE       CURRENTVERSION                                   TARGETVERSION                                             DETAIL
eksa-packages   package.packages.eks.amazonaws.com/my-test   test      8s    installed   0.1.1-6d0e566362d03a00f41d75524d21de5ddb95fd1d   0.1.1-6d0e566362d03a00f41d75524d21de5ddb95fd1d (latest)   

NAMESPACE       NAME                                                  STATE
eksa-packages   packagebundle.packages.eks.amazonaws.com/v1-21-1001   active

NAMESPACE       NAME                                                                                 STATE          DETAIL
eksa-packages   packagebundlecontroller.packages.eks.amazonaws.com/eksa-packages-bundle-controller   disconnected  
```

### CODE REVIEW

File: api/testdata/test.yaml [hunk 1]
```diff
@@ -4,7 +4,7 @@ metadata:
   name: my-test
   namespace: eksa-packages
 spec:
-  packageName: Test
+  packageName: test
```
Comment (ewollesen) [inline api/testdata/test.yaml:7]: Did you test this? Why do I feel like this has to match line 9 exactly (ie case-sensitively)?

Comment (TerryHowe) [reply api/testdata/test.yaml:7]: Yeh, I tested, the comparison is case insensitive line 9 should technically be lower case too

Comment (ewollesen) [reply api/testdata/test.yaml:7]: 🤷🏻 ok If either works, I don't see why we're changing things, but if it works, I can't object.

/lgtm

**Review by ewollesen — COMMENTED** [review]
/lgtm

**Review by lewisdiamond — APPROVED** [review]

**Pull request merged**